Neos.EventSourcedNeosAdjustments/Classes/Ui/Domain/Model/Changes/Property.php

```diff
@@ -202,12 +202,26 @@ public function apply()
             );
 
             // TODO: Make changing the node type a separated, specific/defined change operation.
-            if ($propertyName === '_nodeType') {
-                throw new \Exception("TODO FIX");
-                $nodeType = $this->nodeTypeManager->getNodeType($value);
-                $node = $this->changeNodeType($node, $nodeType);
-            } elseif ($propertyName{0} === '_') {
-                if ($propertyName === '_hidden') {
+            if ($propertyName{0} !== '_' || $propertyName === '_hiddenInIndex') {
```

Comment:
Can we asume that `$propertyName` is not an empty string here?

Reply:
yeah, I think so. Property Name is never allowed to be empty